package.json

```diff
@@ -24,6 +24,7 @@
     "react-router-dom": "^5.2.0",
     "react-scripts": "^1.0.10",
     "react-sticky-footer": "0.1.0-rc3",
+    "react-tooltip":"0.0.1",
```

Comment:
Why react-tooltip version 0.0.1 instead of the latest version? Did you use `npm install react-tooltip --save` to add it to the project?

Using version 0.0.1 leads to this compilation error, which isn't present using the latest version:

```
./node_modules/react-tooltip/index.js
Module parse failed: Unexpected token (75:6)
You may need an appropriate loader to handle this file type.
|     })
|     return (
|       <span className={toolTipClass} style={style} data-id="tooltip">{this.state.placeholder}</span>
|     )
|   }
```